dashboard/public/google3799398f478c3974.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google3799398f478c3974.html
```

Comment:
Not blocking, but for my own learning why is this in dashboard?

Reply:
Looking to also add studio.code.org on search console!